packages/core/src/App/Containers/Layout/header/trading-hub-header.jsx

```diff
@@ -82,6 +83,32 @@ const ShowNotifications = ({ is_notifications_visible, notifications_count, togg
     );
 };
 
+const ShowAccountSwitcher = ({
```

Comment:
@aizad-deriv What's the point of `ShowAccountSwitcher`? We can just use `AccountInfo` itself 👀